cross_compile/Dockerfile_workspace

```diff
@@ -39,14 +39,16 @@ RUN sh -c 'echo "deb [arch=amd64,arm64] http://packages.ros.org/ros2/ubuntu `lsb
     > /etc/apt/sources.list.d/ros2-latest.list'
 
 # ROS2 dependencies
-RUN apt update && apt install -y \
+RUN apt-get update && apt-get install -y \
     build-essential \
     cmake \
     git \
     python3-pip \
     python-rosdep \
     wget
 
+RUN apt-get install -y symlinks
```

Comment:
Could you document why it's needed?

Reply:
Needed [here](https://github.com/ros-tooling/cross_compile/blob/20f355a9111992d07573d44c0ee2909e8a2fa78d/cross_compile/Dockerfile_workspace#L112)

Reply:
Could you document why we need this?